consumers/campaigner/main.go

```diff
@@ -73,14 +81,26 @@ func (h *MessageHandler) HandleMessage(m *nsq.Message) (err error) {
 		return err
 	}
 
-	if campaign.Status != entities.StatusDraft {
-		logEntry.WithError(err).Errorf("potentially duplicate message: campaign status is '%s', it should be 'draft'", campaign.Status)
-		return nil
+	if campaign.EventID == nil {
+		logEntry.WithError(err).Error("event id not found")
+		return err
```

Comment:
If the event id is not found, I'm not sure if we retry the message here, since it will just go in a loop.

Reply:
There is a small chance that the fetch from the consumer will be faster than the insert in the handler but yeah I am not sure if we should loop over it

Reply:
The event id must be present before we publish a message to the consumer. We should find another solution to this, maybe change the handler?